Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -491,6 +491,13 @@ Function Main {
         }
         #PROVISION VMS FOR LISA WILL ENABLE ROOT USER AND WILL MAKE ENABLE PASSWORDLESS AUTHENTICATION ACROSS ALL VMS IN SAME HOSTED SERVICE.
         Provision-VMsForLisa -allVMData $AllVMData -installPackagesOnRoleNames "none"
+        $sts = Check-Modules
+        $sts = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
```

Comment:
I recommend to avoid using -ignoreLinuxExitCode. 
A better approach is to download the log file and check the contents.